src/ralph/models/xapi/fields/unnested_objects.py

```diff
@@ -89,7 +89,7 @@ class ActivityObjectField(BaseModelWithConfig):
         definition (dict): See ObjectDefinitionField.
     """
 
-    id: IRI
+    id: IRI = IRI()
```

Comment:
It seems some tests fail because the `IRI()` creates an empty string that is not considered a valid value.
Why do we need a default value for the object ID if it is a required field?